package.json

```
@@ -17,39 +17,32 @@
   },
   "dependencies": {
     "asar": "^1.0.0",
-    "bluebird": "^3.3.4",
     "debug": "^2.2.0",
-    "fs-extra": "^2.1.2",
+    "fs-extra": "^7.0.1",
     "lodash.template": "^4.2.2",
+    "pify": "^4.0.1",
     "temp": "^0.8.3"
   },
   "devDependencies": {
-    "ava": "^0.13.0",
-    "babel-cli": "^6.6.5",
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.3.3",
+    "@babel/plugin-transform-async-to-generator": "^7.2.0",
+    "@babel/polyfill": "^7.2.5",
+    "@babel/preset-env": "^7.3.1",
+    "@babel/register": "^7.0.0",
+    "ava": "^1.2.1",
     "babel-eslint": "^6.0.2",
-    "babel-plugin-transform-async-to-module-method": "^6.7.0",
-    "babel-plugin-transform-runtime": "^6.6.0",
-    "babel-preset-es2015-node4": "^2.0.3",
-    "babel-preset-stage-0": "^6.5.0",
-    "babel-register": "^6.7.2",
     "eslint": "^2.4.0"
   },
   "engines": {
     "node": ">=0.4.0"
```

I don't even know why it's `0.4.0` in the first place...

```suggestion
     "node": ">=6.0.0"
```